src/05-derivatives/05-diffusion-derivatives.md

```diff
@@ -0,0 +1,467 @@
+    -   `StaticDiffusivity` : value
+
+-   `FORECAST` :
+
+    -   `Sphere` : value
+    -   `DecAlg` : value
+    -   `LambdaLb` : value
+    -   `SphericalHarmonicsOrder` : value
+
+-   `fwDTI` :
+
+    -   `FitMethod` : {"WLS","NLLS"}
+
+-   `BEDPOSTX` :
+
+    -   `NFibers` : value
+    -   `Fudge` : value
+    -   `BurnIn` : value
+    -   `NJumps` : value
+    -   `SampleEvery` : value
+    -   `Model` : {`monoexponential`,`multiexponentialStick`,`multiexponentialZeppelin`}
+
+Examples:
+
+A Diffusion Tensor fit:
+
+```Text
+my_diffusion_pipeline/
+    sub-01/
+        dwi/
+            sub-01_space-T1w_desc-WLS_model-DTI_diffmodel.nii.gz
+            sub-01_space-T1w_desc-WLS_model-DTI_diffmodel.json
+```
+
+Contents of JSON file:
+
+```JSON
+{
+    "Parameters": {
+        "FitMethod": "WLS"
+    }
+}
+```
+
+A multi-shell, multi-tissue Constrained Spherical Deconvolution fit:
+
+```Text
+my_diffusion_pipeline/
+    sub-01/
+        dwi/
+            sub-01_model-CSD_desc-WM_diffmodel.nii.gz
+            sub-01_model-CSD_desc-WM_diffmodel.json
+            sub-01_model-CSD_desc-GM_diffmodel.nii.gz
+            sub-01_model-CSD_desc-GM_diffmodel.json
+            sub-01_model-CSD_desc-CSF_diffmodel.nii.gz
+            sub-01_model-CSD_desc-CSF_diffmodel.json
+```
+
+Contents of JSON file (using "`_tissue-WM`" as example):
+
+```JSON
+{
+    "Shells": [ 0, 1000, 2000, 3000 ],
+    "Parameters": {
+        "Tissue": "White matter",
+        "SphericalHarmonicDegree": 8,
+        "ResponseFunctionZSH": [ [ 15335 0 0 0 0 0 ],
+                                 [ 7688 -2766 597 -82 10 -5 ],
+                                 [ 5258 -2592 1024 -270 48 -11 ],
+                                 [ 4226 -2193 1124 -436 119 -19 ] ]
+        "SphericalHarmonicBasis": "MRtrix3",
+        "NonNegativityConstraint": "hard"
+    }
+}
+```
+
+A NODDI fit:
+
+```Text
+my_diffusion_pipeline/
+    sub-01/
+        dwi/
+            sub-01_model-NODDI_parameter-ICVF_diffmodel.nii.gz
+            sub-01_model-NODDI_parameter-OD_diffmodel.nii.gz
+            sub-01_model-NODDI_parameter-ISOVF_diffmodel.nii.gz
+            sub-01_model-NODDI_parameter-direction_diffmodel.nii.gz
+            sub-01_model-NODDI_diffmodel.json
+```
+
+## Model-derived maps (output parameters)
+
+Models output maps of estimated parameters. Commonly one parameter is estimated
+per voxel; we call these maps of parameters. Two types of maps are described in
+this specification: scalar and vector maps. Scalar maps are saved as 3D
+.nii/nii.gz files. Vector maps are saved as 4D .nii/nii.gz files. Below the
+specification for the naming of the files and a the list of currently accepted
+fields for the maps.
+
+```Text
+<pipeline_name>/
+    sub-<participant_label>/
+        dwi/
+            <source_keywords>[_space-<space>]_model-<label>[_desc-<label>]_<parameter>.nii[.gz]
+```
+
+### Scalar maps
+
+These maps are saved 3D NIfTI files (x,y,z). Some examples of such maps are as follows:

```

Comment:
I'm not sure why models are all represented by a unique suffix (`diffmodel`) but scalars get their own suffix. Just pointing at something we might want to discuss further.